publicationiq.json

```diff
@@ -289,6 +289,41 @@
               "packagename"
             ]
           },
+          {
+            "name": "includeFacets",
+            "in":"query",
+            "description":"Specify **true** if search is required to include facet information, **false** otherwise.",
+            "required":true,

```

Comment:
<img width="1167" alt="Screenshot 2024-08-13 at 1 54 21 PM" src="https://github.com/user-attachments/assets/516ea257-54d0-493f-b83c-a491e64ebb4e">

This is in compliance with the story ACs, but wondering why this is being marked as a required parameter when it is optional on PF API.

Reply:
I will swap it to optional here. Not sure where the required came from, I know it's marked as required on the publications endpoint so maybe that made it into the ACs of this story. 
![image](https://github.com/user-attachments/assets/1fe5a536-b83d-4c52-9281-60d7b3fe6de7)